homeassistant/components/elgato/light.py

```diff
@@ -59,7 +59,14 @@ def __init__(self, coordinator: ElgatoDataUpdateCoordinator) -> None:
         self._attr_unique_id = coordinator.data.info.serial_number
 
         # Elgato Light supporting color, have a different temperature range
-        if self.coordinator.data.settings.power_on_hue is not None:
+        if (
+            self.coordinator.data.info.product_name
+            in (
+                "Elgato Light Strip",
+                "Elgato Light Strip Pro",
+            )
```

Comment:
How about checking for the `hue` or `power_on_hue` property instead?

Reply:
So if we set the light strip color using a color temperature (like in the restore state setting) "hue" can be undefined (shown in the test fixtures btw)

Reply:
Thanks, makes sense. I'll considering moving this into the library over time maybe. For now, this is fine 👍